api/v1/flow/flow.proto

```diff
@@ -306,10 +306,8 @@ enum DebugCapturePoint {
     DBG_CAPTURE_FROM_LB = 5;
     DBG_CAPTURE_AFTER_V46 = 6;
     DBG_CAPTURE_AFTER_V64 = 7;
-    DBG_CAPTURE_PROXY_PRE = 8;
-    DBG_CAPTURE_PROXY_POST = 9;
-    DBG_CAPTURE_SNAT_PRE = 10;
-    DBG_CAPTURE_SNAT_POST = 11;
+    DBG_CAPTURE_SNAT_PRE = 8;
+    DBG_CAPTURE_SNAT_POST = 9;
```

Comment:
this is a breaking change, are we sure we want to re-use the indices?

Reply:
Should I revert the indices back ?

Reply:
Yeah. Given there is no benefit to shifting the indices for proto, I think it's best to keep them at the original values and leave a comment saying `8-11 are deprecated`.

Reply:
pushed the fix for the indices!